### ISSUE TITLE
Support global configuration of log rotation

### ISSUE THREAD

**Issue by vitaly-krugl** [Supervisor/supervisor#585]
Presently, each service's section needs to include its own `stdout_logfile_maxbytes` and `stdout_logfile_backups`.

In our deployments, we make use of 
```
[include]
files = ...
```
to include individual service configurations into a master supervisor.conf for a given product.

We find that the same log rotation configuration is acceptable for all of our services in a given product, but they need to vary between products. For some products, we wish to manage our own log rotation, while in others it's desirable to have supervisord manage log rotation. So, I wanted to configure log rotation globally in each product's supervisor.conf, but there are not settings for it.

It seems to me that many others might be in the same situation: either needing different rotation config between products that share service implementations or simply wanting to configure the same log rotation for all services globally.

**Comment by CTrabant** [comment]
A second vote for this feature.

In addition to the mentioned stdout_logfile_maxbytes and stdout_logfile_backups options I would like to be able to set global versions of all logging related options (e.g. redirect_stderr) and pretty much anything else that goes into a [program] sections.

In my scenario supervisor is managed a number of similar processes and these options are simply redundant in each sections.

**Comment by mikeyg123** [comment]
A third vote
However I'd like to be able to have variables substituted into those variables once a real [program] section is created.  i.e
stdout_logfile = /var/logs/%(program_name)s.log

**Comment by akhiljalagam** [comment]
yeah, something like in https://github.com/ochinchina/supervisord with option `[program-default]` would be very useful
